Document/0x04a-Mobile-App-Taxonomy.md

```diff
@@ -6,10 +6,11 @@ Also, this guide uses the term "app" as a general term which refers to any kind
 
 We place mobile apps into four categories:
 
-- [Native Apps](#native-apps)
-- [Web Apps](#web-apps)
-- [Hybrid Apps](#hybrid-apps)
-- [Progressive Web Apps](#progressive-web-apps)
+- [Mobile Application Taxonomy](#mobile-application-taxonomy)
```

Comment:
As we want to highlight the 4 categories, there is no need to add again the Mobile App Taxonomy. 
```suggestion
```